src/Examples/Flowsheets/methanol_synthesis.ipynb

```diff
@@ -161,7 +161,7 @@
    "metadata": {},
    "source": [
     "## 3.2 Build and Solve Flowsheet\n",
-    "The methanol flowsheet methods are called sequentially below, following the workflow contained in the ```main()``` method in ```idaes/models/flowsheets/methanol_flowsheet.py```. First, let's set the solver options. IDAES contains a default solver `get_solver` which calls IPOPT using standard settings, and we set an iteration cap of 100 to catch nonconverging solver runs."
+    "The methanol flowsheet methods are called sequentially below, following the workflow contained in the ```main()``` method in ```flowsheet.py```. First, let's set the solver options. IDAES contains a default solver `get_solver` which calls IPOPT using standard settings, and we set an iteration cap of 100 to catch nonconverging solver runs."
```

Comment:
I'm wondering what would be the best way to refer to this file now that it lives in a different directory. One method that's a bit roundabout but not likely to break (at least in the short term) would be a link with the full path to the file, pointing to the file on GitHub:

```suggestion
    "The methanol flowsheet methods are called sequentially below, following the workflow contained in the ```main()``` method in [pkg.idaes_examples.common.methanol.flowsheet](https://github.com/IDAES/examples-pse/tree/main/pkg/idaes_examples/common/methanol/flowsheet.py). First, let's set the solver options. IDAES contains a default solver `get_solver` which calls IPOPT using standard settings, and we set an iteration cap of 100 to catch nonconverging solver runs."
```

This is a bit verbose, but at least it makes easier to search and replace when or if we want to use a different scheme.

Reply:
I like this suggestion, thanks. I'll update the other script references in this manner as well.

Git somehow didn't check-in setup.py (I thought it did) so I'll fix that and the `get_solver` issue to get the tests passing.